automation_py/automations/gmail_creator/core.py

```
@@ -271,3 +382,315 @@ def verify_account():
+
+            # Salvar o arquivo atualizado
+            with open(credentials_path, "w") as file:
+                json.dump(existing_accounts, file, indent=4)
+
+            logger.info(
+                f"[OK] Arquivo de credenciais salvo com sucesso: {credentials_path}")
+
+            # Verificar se o arquivo foi salvo corretamente
+            if os.path.exists(credentials_path) and os.path.getsize(credentials_path) > 0:
+                logger.info(
+                    f"[OK] Verificação: arquivo de credenciais existe e tem conteúdo")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)